osbs/api.py

```diff
@@ -395,13 +393,21 @@ def _create_build_config_and_build(self, build_request):
             logger.debug('Changed parent ImageStreamTag? %s', changed_ist)
 
         if triggers:
-            existing_bc['spec']['triggers'] = triggers
-            self.os.update_build_config(build_config_name, json.dumps(existing_bc))
```

Comment:
TBH I don't see how switching to patch makes this better. In either case, it's completely irrelevant to this PR.